Orange/canvas/document/usagestatistics.py

```diff
@@ -105,6 +105,31 @@ def log_node_added(self, widget_name, extended_widget=None):
     def set_node_type(self, addition_type):
         self.__node_addition_type = addition_type
 
+    def send_statistics(self):
+        def store_data(d):
+            with open(statistics_path, 'w') as f:
+                json.dump(d, f)
+        if release and config.settings()["error-reporting/send-statistics"]:
+            if os.path.isfile(statistics_path):
+                with open(statistics_path) as f:
+                    data = json.load(f)
+            else:
+                data = []
```

Comment:
Why else here? Why would we want to send empty data?

Reply:
I moved the sending part inside if os.path.isfile(statistics_path) for the rare case, when there is no file (e.g. directly after installation).